case-lib/lib.sh

```diff
@@ -252,6 +252,19 @@ func_lib_get_tplg_path()
     return 0
 }
 
+func_nocodec_mode()
+{
+    uname -r |grep -q "nocodec"
+}
+
+func_upload_wav_file()
+{
+    local dir=$1
+    local file=$2
```

Comment:
While double quotes are technically not required for assignments it's safer and good role modelling to just always quote.